.github/workflows/go.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
     - name: before_install from Travis
       run: go get golang.org/x/tools/cmd/goimports && go install golang.org/x/tools/cmd/goimports
-    - run: go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest
```

Comment:
As a general rule I tend to lock the version number (at least the major version) for any tool in use. Otherwise subtle changes can break the build for no good reason.